contrib/win32/win32compat/misc.c

```diff
@@ -455,20 +455,55 @@ w32_chown(const char *pathname, unsigned int owner, unsigned int group) {
 	return -1;
 }
 
+void
+UnixTimeToFileTime(ULONG t, LPFILETIME pft) {
```

Comment:
need to follow rest of codes naming convention of routine names (all lower case, words separated by "_")